#### Find yourself a data file from an open data repository. 
#### Select an interesting table of data and convert it into a CSV if necessary. 
#### Then load it into a Python program and examine the data that comes in. 
#### Can you do some basic statistics on it with numpy?

In [2]:
import csv
with open('Data/COVID-19_cases__per_100_000_population__in_Alberta_and_by_zone.csv') as csvfile:
    reader = csv.reader(csvfile, delimiter=',')
    list = []
    for column in reader:
        list.append(column[1])

list = list[1:]

print(list)
    

['4111.5', '4212.4', '3135.7', '4578.4', '3909.1', '3180.9']


Data Source

[https://data.edmonton.ca/Community-Services/COVID-19-in-Alberta-Current-cases-by-local-geograp/ix8f-s9xp](https://data.edmonton.ca/Community-Services/COVID-19-in-Alberta-Current-cases-by-local-geograp/ix8f-s9xp)


This data is from the province of Alberta in Canada describing the current COVID-19 cases by local geographic area.



In [18]:
import pandas as pd
# df = pd.read_csv('Data/COVID-19_cases__per_100_000_population__in_Alberta_and_by_zone.csv')
df = pd.read_csv('Data/COVID-19_in_Alberta__Current_cases_by_local_geographic_area.csv')

#get max and min active cases
max_active = df.loc[df['Active Cases'].idxmax()]['Location']
min_active = df.loc[df['Active Cases'].idxmin()]['Location']

print('Max Active Cases: \n', max_active)
print('Min Active Cases: \n', min_active)

#get max and min total cases
max_total = df.loc[df['Total Cases'].idxmax()]['Location']
min_total = df.loc[df['Total Cases'].idxmin()]['Location']

print('Max Total Cases: \n', max_total)
print('Min Total Cases: \n', min_total)

df.describe()



Max Active Cases: 
 Fort Mcmurray
Min Active Cases: 
 Viking (Beaver County East)
Max Total Cases: 
 Calgary - Upper Ne
Min Total Cases: 
 Swan Hills (South Big Lakes County)


Total Cases  Active Cases  Recovered Cases      Deaths
count    131.000000    131.000000       131.000000  131.000000
mean    1309.480916    138.847328      1155.183206   15.450382
std     1679.773063    167.855087      1524.566182   24.683602
min        9.000000      0.000000         9.000000    0.000000
25%      189.500000     23.000000       160.000000    1.000000
50%      609.000000     76.000000       513.000000    6.000000
75%     1623.500000    204.500000      1459.500000   17.500000
max    11261.000000    833.000000     10410.000000  133.000000

The average amount of **Active Cases** in each region is 139 with a standard deviation of 167.
The max active cases is 833 in Fort Mcmurray and the min is 0. in Viking. Being 0 there is a possibility there are other areas that also have 0.

The average amount of **Total Cases** in each region is 1309 with a standard deviation of 1680.
The max total cases is 11261 in Calgary Upper NE and min is 9 in Swan Hills.


